pkg/ccl/backupccl/backupresolver/targets.go

```diff
@@ -401,7 +401,7 @@ func DescriptorsMatchingTargets(
 			if schemaDesc == nil || !schemaDesc.Public() {
 				if requirePublic {
 					return errors.Wrapf(err, "schema %d was expected to be PUBLIC", id)
-				} else if schemaDesc == nil || !schemaDesc.Offline() {
+				} else if schemaDesc == nil || schemaDesc.Dropped() {
```

Comment:
This change means we will include schema descriptors that are in `ADD` state to the backup (previously, only public and offline ones are included).

Reply:
You should probably update the comment below also.